python/ray/_private/state_api_test_utils.py

```diff
@@ -0,0 +1,284 @@
+import asyncio
```

Comment:
is it possible to move this file out of _private folder?

Reply:
What would be a better place for this? I put it here as `test_utils.py` is here as well.